src/abstractdataframe/reshape.jl

```diff
@@ -31,6 +32,9 @@ that return views into the original data frame.
   each of `measure_vars`
 - `view` : whether the stacked data frame should be a view rather than contain
    freshly allocated vectors.
+- `vareltype` : determines the element type of column `variable_name`. By default
+   a categorical vector is created. If `vareltype=Symbol` it is a vector of `Symbol`,
```

Comment:
```suggestion
   a `CategoricalVector{String}` is created. If `vareltype=Symbol` it is a vector of `Symbol`,
```

Reply:
it is not true for `view=true` as then it is `RepeatedVector`. So I was not sure what to write.
Given this comment do you still think we should apply your change?

Reply:
Ah, right. Then maybe just "a categorical vector of strings" or something like that.

Reply:
fixed